src/coreclr/vm/fcall.h

```diff
@@ -1,91 +1,16 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
-// FCall.H
-//
 
 //
-// FCall is a high-performance alternative to ECall. Unlike ECall, FCall
-// methods do not necessarily create a frame.   Jitted code calls directly
-// to the FCall entry point.   It is possible to do operations that need
-// to have a frame within an FCall, you need to manually set up the frame
-// before you do such operations.
-
-// It is illegal to cause a GC or EH to happen in an FCALL before setting
-// up a frame.  To prevent accidentally violating this rule, FCALLs turn
-// on BEGINGCFORBID, which ensures that these things can't happen in a
-// checked build without causing an ASSERTE.  Once you set up a frame,
-// this state is turned off as long as the frame is active, and then is
-// turned on again when the frame is torn down.   This mechanism should
-// be sufficient to ensure that the rules are followed.
-
-// In general you set up a frame by using the following macros
-
-//      HELPER_METHOD_FRAME_BEGIN_RET*()    // Use If the FCALL has a return value
-//      HELPER_METHOD_FRAME_BEGIN*()        // Use If FCALL does not return a value
-//      HELPER_METHOD_FRAME_END*()
+// FCall is a high-performance call into unmanaged runtime code from managed code.
+// The managed code calls directly to the FCall entry point.  It is possible to do
+// operations that need to have a frame within an FCall, you need to manually set up
+// the frame before you do such operations.
```

Comment:
```suggestion
// The managed code calls directly to the FCall entry point.
```

Should we just drop the second part? Technically you can still establish a frame in FCall but it's only used in a very specialized situations like `IL_Throw` and it's beyond the scope of the macros in this file.